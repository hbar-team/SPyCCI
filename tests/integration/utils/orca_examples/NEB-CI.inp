%pal
  nprocs 8
end

%maxcore 750

! PBE def2-SVP D3BJ
! NEB-CI

%neb
  product "NEB_product.xyz"
  nimages 5
end

* xyzfile 0 1 NEB_reactant.xyz

