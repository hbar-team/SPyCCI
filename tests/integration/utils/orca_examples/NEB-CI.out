
                                 *****************
                                 * O   R   C   A *
                                 *****************

                                            #,                                       
                                            ###                                      
                                            ####                                     
                                            #####                                    
                                            ######                                   
                                           ########,                                 
                                     ,,################,,,,,                         
                               ,,#################################,,                 
                          ,,##########################################,,             
                       ,#########################################, ''#####,          
                    ,#############################################,,   '####,        
                  ,##################################################,,,,####,       
                ,###########''''           ''''###############################       
              ,#####''   ,,,,##########,,,,          '''####'''          '####       
            ,##' ,,,,###########################,,,                        '##       
           ' ,,###''''                  '''############,,,                           
         ,,##''                                '''############,,,,        ,,,,,,###''
      ,#''                                            '''#######################'''  
     '                                                          ''''####''''         
             ,#######,   #######,   ,#######,      ##                                
            ,#'     '#,  ##    ##  ,#'     '#,    #''#        ,####,    ,#,          
            ##       ##  ##   ,#'  ##            #'  '#       #'       ,# #          
            ##       ##  #######   ##           ,######,      #####,      #          
            '#,     ,#'  ##    ##  '#,     ,#' ,#      #,     #,   #      #          
             '#######'   ##     ##  '#######'  #'      '#     '####' #    #          



                #########################################################
                #                        -***-                          #
                #          Department of theory and spectroscopy        #
                #                                                       #
                #                      Frank Neese                      #
                #                                                       #
                #     Directorship, Architecture, Infrastructure        #
                #                    SHARK, DRIVERS                     #
                #        Core code/Algorithms in most modules           #
                #                                                       #
                #        Max Planck Institute fuer Kohlenforschung      #
                #                Kaiser Wilhelm Platz 1                 #
                #                 D-45470 Muelheim/Ruhr                 #
                #                      Germany                          #
                #                                                       #
                #                  All rights reserved                  #
                #                        -***-                          #
                #########################################################


                         Program Version 6.1.0  -  RELEASE   -
                                (GIT: $679e74b$)
                          ($2025-06-10 18:02:51 +0200$)


 With contributions from (in alphabetic order):
[Max-Planck-Institut fuer Kohlenforschung]                               
  Daniel Aravena         : Magnetic Suceptibility                        
  Michael Atanasov       : Ab Initio Ligand Field Theory (pilot matlab implementation)
  Alexander A. Auer      : GIAO ZORA, VPT2 properties, NMR spectrum      
  Ute Becker             : All parallelization in ORCA, NUMFREQ, NUMCALC 
  Giovanni Bistoni       : ED, misc. LED, open-shell LED, HFLD           
  Dmytro Bykov           : pre 5.0 version of the SCF Hessian            
  Marcos Casanova-PÃ¡ez   : Triplet and SCS-CIS(D). UHF-(DLPNO)-IP/EA/STEOM-CCSD. UHF-CVS-IP/STEOM-CCSD   
  Vijay G. Chilkuri      : MRCI spin determinant printing, contributions to CSF-ICE 
  Pauline Colinet        : FMM embedding                                 
  Dipayan Datta          : RHF DLPNO-CCSD density                        
  Achintya Kumar Dutta   : EOM-CC, STEOM-CC                              
  Nicolas Foglia         : Exact transition moments, OPA infrastructure, MCD improvements    
  Dmitry Ganyushin       : Spin-Orbit,Spin-Spin,Magnetic field MRCI      
  Miquel Garcia-Rates    : C-PCM and meta-GGA Hessian, CCSD/C-PCM, Gaussian charge scheme 
  Tiago L. C. Gouveia    : GS-ROHF, GS-ROCIS                             
  Yang Guo               : DLPNO-NEVPT2, F12-NEVPT2, CIM, IAO-localization        
  Andreas Hansen         : Spin unrestricted coupled pair/coupled cluster methods 
  Ingolf Harden          : AUTO-CI MPn and infrastructure                
  Benjamin Helmich-Paris : MC-RPA, TRAH-(SCF,CASSCF), AVAS, COSX integrals, SCF dyn. polar., MC-PDFT, srDFT 
  Lee Huntington         : MR-EOM, pCC                                   
  Robert Izsak           : Overlap fitted RIJCOSX, COSX-SCS-MP3, EOM     
  Riya Kayal             : Wick's Theorem for AUTO-CI, AUTO-CI UHF-CCSDT 
  Emily Kempfer          : AUTO-CI RHF CISDT and CCSDT, approximate NEVPT4
  Christian Kollmar      : KDIIS, OOCD, Brueckner-CCSD(T), CCSD density, CASPT2, CASPT2-K, improved NEVPT2 
  Axel Koslowski         : Symmetry handling                             
  Simone Kossmann        : meta-GGA functionals, TD-DFT gradient, OOMP2, (MP2 Hessian; deprecated post 5.0) 
  Lucas Lang             : DCDCAS, Hyperfine gauge corrections, ICE-SOC+SSC
  Marvin Lechner         : AUTO-CI (C++ implementation), FIC-MRCC        
  Spencer Leger          : CASSCF response                               
  Dagmar Lenk            : GEPOL surface, SMD, ORCA-2-JSON               
  Dimitrios Liakos       : Extrapolation schemes; Compound Job, Property file
  Dimitrios Manganas     : Further ROCIS development; embedding schemes. LFT, Crystal Embedding 
  Dimitrios Pantazis     : SARC Basis sets                               
  Anastasios Papadopoulos: AUTO-CI, single reference methods and gradients 
  Taras Petrenko         : pre 6.0 DFT Hessian and TD-DFT gradient,  ECA, NRVS 
  Petra Pikulova         : Analytic Raman intensities                    
  Peter Pinski           : DLPNO-MP2, DLPNO-MP2 Gradient                 
  Shashank Vittal Rao    : ES-AILFT, MagRelax                            
  Christoph Reimann      : Effective Core Potentials                     
  Marius Retegan         : Local ZFS, SOC                                
  Christoph Riplinger    : Optimizer, TS searches, QM/MM, DLPNO-CCSD(T), (RO)-DLPNO pert. Triples 
  Michael Roemelt        : Original ROCIS implementation, recursive CI coupling coefficients
  Masaaki Saitow         : Open-shell DLPNO-CCSD energy and density      
  Barbara Sandhoefer     : DKH picture change effects                    
  Yorick L. A. Schmerwitz: GMF and freeze-and-release deltaSCF, NEB S-IDPP initial path                    
  Kantharuban Sivalingam : CASSCF convergence/infrastructure, NEVPT2, NEVPT3, NEVPT4(SD), FIC-MRCI and CEPA variants
  Bernardo de Souza      : ESD, SOC TD-DFT                               
  Georgi L. Stoychev     : AutoAux, RI-MP2 NMR, DLPNO-MP2 response, X2C  
  Van Anh Tran           : RI-MP2 g-tensors                              
  Willem Van den Heuvel  : Paramagnetic NMR                              
  Zikuan Wang            : NOTCH, Electric field optimization            
  Frank Wennmohs         : Technical directorship and infrastructure     
  Hang Xu                : AUTO-CI-Response properties                   
                                                                         
[FACCTs GmbH]                                                            
  Markus Bursch, Nicolas Foglia, Miquel Garcia-Rates, Ingolf Harden, Hagen Neugebauer, Anastasios Papadopoulos, 
  Christoph Riplinger, Bernardo de Souza, Georgi L. Stoychev                                                    
                                                                 
  APM, various basis sets, CI-OPT, improved COSX, DLPNO-Multilevel,           
  DOCKER, DRACO, updates on ESD, Fragmentator, GOAT, IRC, LR-CPCM, L-BFGS, MBIS, meta-GGA TD-DFT gradient, ML-optimized integration grids, 
  MM, NACMEs, nearIR, NEB, NEB-TS, NL-DFT gradient (VV10), 2- and 3-layer-ONIOM, interface openCOSMO-RS, QMMM,   
  Crystal-QMMM, RESP, rigid body optimization, SF, symmetry and pop. for TD-DFT, various functionals, SOLVATOR                                   
                                                                         
[Other institutions]                                                     
  V. Asgeirsson          : NEB                                           
  Christoph Bannwarth    : sTDA-DFT, sTD-DFT, PBEh-3c, B97-3c, D3        
  Giovanni Bistoni       : ETS/NOCV, ADLD/ADEX, COVALED                  
  Martin Brehm           : Molecular dynamics                            
  Ronald Cardenas        : ETS/NOCV                                      
  Martina Colucci        : COVALED                                       
  Sebastian Ehlert       : rSCAN, r2SCAN, r2SCAN-3c, D4, dhf basis sets  
  Marvin Friede          : D4 for Fr, Ra, Ac-Lr                          
  Lars Goerigk           : TD-DFT with DH, B97 family of functionals     
  Stefan Grimme          : VdW corrections, initial TS optimization, DFT functionals, gCP, sTDA/sTD-DF 
  Waldemar Hujo          : DFT-NL                                        
  H. Jonsson             : NEB                                           
  Holger Kruse           : gCP                                           
  Marcel Mueller         : wB97X-3c, vDZP basis set                      
  Hagen Neugebauer       : wr2SCAN, Native XTB                           
  Gianluca Regni         : ADLD/ADEX                                     
  Tobias Risthaus        : pre 6.0 range-separated hybrid DFT and stability analysis
  Lukas Wittmann         : regularized MP2, r2SCAN double-hybrids, wr2SCAN        
                                                                         
We gratefully acknowledge several colleagues who have allowed us to      
interface, adapt or use parts of their codes:                            
  Ed Valeev, F. Pavosevic, A. Kumar             : LibInt (2-el integral package), F12 methods 
  Garnet Chan, S. Sharma, J. Yang, R. Olivares  : DMRG                   
  Ulf Ekstrom                                   : XCFun DFT Library      
  Mihaly Kallay                                 : mrcc  (arbitrary order and MRCC methods)
  Frank Weinhold                                : gennbo (NPA and NBO analysis)           
  Simon Mueller                                 : openCOSMO-RS                            
  Christopher J. Cramer and Donald G. Truhlar   : smd solvation model                     
  S Lehtola, MJT Oliveira, MAL Marques          : LibXC Library                           
  Liviu Ungur et al                             : ANISO software                          


 Your calculation uses the libint2 library for the computation of 2-el integrals
 For citations please refer to: http://libint.valeyev.net

 Your ORCA version has been built with support for libXC version: 7.0.0
 For citations please refer to: https://libxc.gitlab.io

 This ORCA versions uses:
   CBLAS   interface :  Fast vector & matrix operations
   LAPACKE interface :  Fast linear algebra routines
   SCALAPACK package :  Parallel linear algebra routines
   Shared memory     :  Shared parallel matrices
   BLAS/LAPACK       :  OpenBLAS 0.3.29  USE64BITINT DYNAMIC_ARCH NO_AFFINITY Haswell SINGLE_THREADED
        Core in use  :  Haswell
   Copyright (c) 2011-2014, The OpenBLAS Project


  ***********************************
  * Starting time: Fri Sep 26 21:04:39 2025
  * Host name:     Host-001
  * Process ID:    60757
  * Working dir.:  /home/ppravatto/Documenti/Esperimenti/ChimicaComputazionale/SPyCCI/NEB_reactant_NEB_product_rsfxu6yq_orca_PBE_def2-SVP_vacuum_NEB-CI
  ***********************************

NOTE: MaxCore=750 MB was set to SCF,MP2,MDCI,CIPSI,MRCI,RASCI and CIS
      => If you want to overwrite this, your respective input block should be placed after the MaxCore statement


***************************************
The coordinates will be read from file: NEB_reactant.xyz
***************************************


Your calculation utilizes the atom-pairwise dispersion correction
with the Becke-Johnson damping scheme (D3BJ)


Your calculation utilizes a Nudged-Elastic-Band implementation
by V. Ãsgeirsson, C. Riplinger & H. JÃ³nsson
Please cite in your paper:
Ãsgeirsson, V.; Birgisson, B. O.; Bjornsson, R.; Becker, U.; Neese, F.; Riplinger, R.;
JÃ³nsson H. J. Chem. Theory Comput. 2021, 17, 4929-4945
   

Warning: RI is on but no J-basis has been assigned. Assigning Def2/J (nothing to worry about!)
================================================================================

----- Orbital basis set information -----
Your calculation utilizes the basis: def2-SVP
   F. Weigend and R. Ahlrichs, Phys. Chem. Chem. Phys. 7, 3297 (2005).

----- AuxJ basis set information -----
Your calculation utilizes the auxiliary basis: def2/J
   H-Rn: F. Weigend, Phys. Chem. Chem. Phys. 8, 1057 (2006).
   Fr-Lr: K. Eichkorn, F. Weigend, O. Treutler, R. Ahlrichs; Theor. Chem. Acc. 97, 119 (1997).

================================================================================
                                        WARNINGS
                       Please study these warnings very carefully!
================================================================================


WARNING: Geometry Optimization
  ===> : Switching off AutoStart
         For restart on a previous wavefunction, please use MOREAD

================================================================================
                                       INPUT FILE
================================================================================
NAME = input.inp
|  1> %pal
|  2>   nprocs 8
|  3> end
|  4> 
|  5> %maxcore 750
|  6> 
|  7> ! PBE def2-SVP D3BJ
|  8> ! NEB-CI
|  9> 
| 10> %neb
| 11>   product "NEB_product.xyz"
| 12>   nimages 5
| 13> end
| 14> 
| 15> * xyzfile 0 1 NEB_reactant.xyz
| 16> 
| 17> 
| 18>                          ****END OF INPUT****
================================================================================


--------------------------------------------------------------------------------
                        Nudged Elastic Band Calculation
--------------------------------------------------------------------------------

Number of images (incl. end points)     ....  7
Number of intermediate images           ....  5
Number of optimized images              ....  5
Optimization of end points before NEB   ....  NO
Use existing gbw files for MO input     ....  NO
Number of atoms                         ....  7
Number of (active) degrees of freedom   ....  21
Constrained atoms                       ....  None

-----------------------------------------
  REACTANT (ANGSTROEM)
-----------------------------------------
C     -4.283709     1.625615    -0.002642 
C     -2.974068     0.889792     0.008208 
H     -3.024283     0.063020     0.735501 
H     -2.144976     1.563000     0.255117 
H     -2.807253     0.423646    -0.976643 
O     -4.416673     2.808623     0.230366 
H     -5.177487     0.992025    -0.249898 

-----------------------------------------
  PRODUCT (ANGSTROEM)
-----------------------------------------
C     -4.278583     1.484518     0.033611 
C     -3.070429     0.911286     0.052179 
H     -2.193945     1.412820     0.468350 
H     -3.806505     3.140723     0.865082 
H     -2.937113    -0.087695    -0.356607 
O     -4.609602     2.722461     0.502765 
H     -5.158632     0.988607    -0.380772 



----------------------
NEB settings
----------------------
Method type                             ....  climbing image
Threshold for climbing image            ....  2.00e-02 Eh/Bohr 
Free endpoints                          ....  off 
Tangent type                            ....  improved 
Number of intermediate images           ....  5 
Number of images free to move           ....  5 
Spring type for image distribution      ....  distance between adjacent images 
Spring constant                         ....  energy weighted (0.0100 -to- 0.1000) Eh/Bohr^2 
Spring force perp. to the path          ....  none 
Generation of initial path              ....  image dependent pair potential 
Initial path via TS guess               ....  off 

Minimization of RMSD: Rotation          ....  always 
Minimization of RMSD: Translation type  ....  centroid 
Center fixed to origin                  ....  true 
Remove external force                   ....  true 

Reparametrization of the path           ....  off 
Convergence thresholds:
Convergence monitored for               ....  all images
Scaling factor                          ....  10.00

Convergence parameters for regular images:
               Max(|Fp|)                ....  5.00e-03 Eh/Bohr
               RMS(Fp)                  ....  2.50e-03 Eh/Bohr

Convergence parameters for climbing image:
               Max(|F|)                 ....  5.00e-04 Eh/Bohr
               RMS(F)                   ....  2.50e-04 Eh/Bohr
Optimization method:
Method                                  ....  L-BFGS
Max. iterations                         ....  500 
Step size                               ....  1.00 
Maximum allowed step size               ....  0.10 Bohr

LBFGS parameters: 
               Memory                   ....  20 
               Initial step size        ....  0.0010 
               Estimate curvature       ....  YES 
               Reset on maxmove         ....  YES 
               Reparam. on reset        ....  NO 
Generation of  the initial path:
Minimize RMSD between reactant and product configurations     .... done
       RMSD before minimization         ....  1.2396 Bohr
       RMSD after minimization          ....  0.9888 Bohr
Performing linear interpolation         ....  done
Generating initial path 1 out of 129 -> Success!
Displacement along initial path:              4.6915 Bohr
Writing initial trajectory to file      ....  input_initial_path_trj.xyz



----------------------
NEB OPTIMIZATION      
----------------------

Following output files will be written:
Energy profile during opt. written to    ....  input.interp
Optimization log-file                    ....  input.NEB.log
Current trajectory will be written to    ....  input_MEP_trj.xyz
       and to allxyz file                ....  input_MEP.allxyz
       and individual images to xyz file ....  input_im{NIm}.xyz
Trajectory history will be written to    ....  input_MEP_ALL_trj.xyz
Converged CI will be written to          ....  input_NEB-CI_converged.xyz

Single point energy and gradient calculation of reactant and
product configurations (have to be performed only once)
Properties of fixed NEB end points:
               Reactant:
                         E               ....   -153.531199 Eh
                         RMS(F)          ....      0.003160 Eh/Bohr
                         MAX(|F|)        ....      0.004948 Eh/Bohr
      Warning: Reactant seems to be less converged than convergence tolerances.
               You might want to use free end NEB or preopt_ends true.
               Product:
                         E               ....   -153.513746 Eh
                         RMS(F)          ....      0.004489 Eh/Bohr
                         MAX(|F|)        ....      0.012592 Eh/Bohr
      Warning: Product seems to be less converged than convergence tolerances.
               You might want to use free end NEB or preopt_ends true.



Starting iterations:

Optim.  Iteration  HEI  E(HEI)-E(0)  max(|Fp|)   RMS(Fp)    dS
Switch-on CI threshold               0.020000 
   LBFGS     0      3    0.132884    0.077186   0.019692  4.6915       
   LBFGS     1      3    0.128124    0.066819   0.016736  4.7117       
   LBFGS     2      3    0.125319    0.058717   0.014239  4.8160       
   LBFGS     3      3    0.121793    0.038029   0.011002  4.8085       
   LBFGS     4      3    0.119760    0.046348   0.010854  4.8342       
   LBFGS     5      3    0.115177    0.048779   0.010555  4.9072       
   LBFGS     6      3    0.111757    0.048722   0.010167  4.9373       
   LBFGS     7      3    0.110006    0.025226   0.006903  4.9493       
   LBFGS     8      3    0.104218    0.022404   0.006441  5.0415       
   LBFGS     9      3    0.102929    0.028030   0.006753  5.0637       
   LBFGS    10      3    0.102086    0.020660   0.005523  5.0877       
   LBFGS    11      3    0.099484    0.018735   0.005172  5.2203       

Image  3 will be converted to a climbing image in the next iteration (max(|Fp|) < 0.0200) 

Optim.  Iteration  CI   E(CI)-E(0)   max(|Fp|)   RMS(Fp)    dS     max(|FCI|)   RMS(FCI)
Convergence thresholds               0.005000   0.002500            0.000500    0.000250 
   LBFGS    12      3    0.099013    0.017027   0.004431  5.2434    0.011301    0.004732       
   LBFGS    13      3    0.098745    0.009123   0.002673  5.2512    0.006300    0.003214       
   LBFGS    14      3    0.098310    0.011848   0.002917  5.2849    0.005254    0.002597       
   LBFGS    15      3    0.097503    0.020199   0.004274  5.3798    0.007723    0.003621       
   LBFGS    16      3    0.097097    0.026183   0.005627  5.5396    0.009691    0.004677       
   LBFGS    17      3    0.096905    0.008799   0.002017  5.5845    0.004987    0.002214       
   LBFGS    18      3    0.096925    0.002595   0.000895  5.5601    0.003266    0.001224       
   LBFGS    19      3    0.096903    0.003140   0.000903  5.5574    0.002921    0.001051       
   LBFGS    20      3    0.096791    0.007989   0.001407  5.5613    0.002999    0.001345       
   LBFGS    21      3    0.096645    0.008269   0.001600  5.5699    0.004268    0.001676       
   LBFGS    22      3    0.096617    0.003816   0.000887  5.5801    0.003825    0.001480       
   LBFGS    23      3    0.096685    0.003722   0.000806  5.5899    0.001619    0.000672       
   LBFGS    24      3    0.096687    0.002817   0.000797  5.5832    0.001411    0.000647       
   LBFGS    25      3    0.096669    0.001740   0.000537  5.5761    0.001396    0.000668       
   LBFGS    26      3    0.096678    0.001708   0.000469  5.5669    0.000810    0.000419       
   LBFGS    27      3    0.096679    0.001996   0.000450  5.5659    0.000605    0.000246       
   LBFGS    28      3    0.096676    0.001106   0.000273  5.5694    0.000553    0.000307       
   LBFGS    29      3    0.096678    0.000621   0.000174  5.5703    0.000426    0.000191       

                                .--------------------.
          ----------------------| CI-NEB convergence |-------------------------
          Item                value                   Tolerance       Converged
          ---------------------------------------------------------------------
          RMS(Fp)             0.0001736055            0.0025000000      YES
          MAX(|Fp|)           0.0006206063            0.0050000000      YES
          RMS(FCI)            0.0001914600            0.0002500000      YES
          MAX(|FCI|)          0.0004262212            0.0005000000      YES
          ---------------------------------------------------------------------

The elastic band and climbing image have converged successfully to a MEP in 29 iterations!


                    *********************H U R R A Y*********************
                    ***        THE NEB OPTIMIZATION HAS CONVERGED     ***
                    *****************************************************



---------------------------------------------------------------
                         PATH SUMMARY              
---------------------------------------------------------------
All forces in Eh/Bohr.

Image Dist.(Ang.)    E(Eh)   dE(kcal/mol)  max(|Fp|)  RMS(Fp)
  0     0.000    -153.53120      0.00       0.00506   0.00313
  1     0.904    -153.49962     19.82       0.00038   0.00019
  2     1.212    -153.45539     47.57       0.00050   0.00021
  3     1.449    -153.43452     60.67       0.00036   0.00017 <= CI
  4     1.727    -153.45964     44.91       0.00041   0.00018
  5     2.120    -153.49289     24.04       0.00062   0.00026
  6     2.948    -153.51375     10.95       0.01258   0.00447

Straight line distance between images along the path:
        D( 0- 1) =   0.9041 Ang.
        D( 1- 2) =   0.3078 Ang.
        D( 2- 3) =   0.2367 Ang.
        D( 3- 4) =   0.2786 Ang.
        D( 4- 5) =   0.3929 Ang.
        D( 5- 6) =   0.8276 Ang.

---------------------------------------------------------------
               INFORMATION ABOUT SADDLE POINT     
---------------------------------------------------------------

Climbing image                            ....  3 
Energy                                    ....  -153.43452080 Eh
Max. abs. force                           ....  4.2622e-04 Eh/Bohr

-----------------------------------------
  SADDLE POINT (ANGSTROEM)
-----------------------------------------
C     -0.663605     0.280407    -0.097089 
C      0.509098    -0.512019    -0.171236 
H      0.838551    -0.860990     0.828273 
H      0.745121     0.864476     0.389865 
H      0.695841    -1.244670    -0.974399 
O     -0.456080     1.365632     0.565600 
H     -1.668927     0.107163    -0.541014 

-----------------------------------------
  FORCES (Eh/Bohr)
-----------------------------------------
C      0.000224    -0.000025     0.000152 
C     -0.000131     0.000426     0.000401 
H     -0.000051     0.000029     0.000029 
H      0.000173    -0.000205    -0.000292 
H     -0.000132    -0.000073    -0.000216 
O     -0.000156    -0.000040     0.000135 
H      0.000072    -0.000113    -0.000209 

-----------------------------------------
  UNIT TANGENT
-----------------------------------------
C      0.108718    -0.152805    -0.130256 
C     -0.174007    -0.292016    -0.106206 
H      0.154206     0.145005    -0.067710 
H     -0.401771     0.530825     0.354519 
H     -0.090282    -0.209574    -0.051558 
O      0.357409    -0.094779     0.044409 
H      0.045727     0.073345    -0.043197 
 => Unit tangent is an approximation to the TS mode at the saddle point
----------
STATISTICS
----------

For the individual parts of the NEB, NEB-CI or NEB-TS run:

Number of iterations:
     NEB                               ... 30  100.0%
Number of SCF / gradient calculations:
     NEB                               ... 177  100.0%

----------
 TIMINGS
----------

For the individual parts of the NEB, NEB-CI or NEB-TS run:

Total                                  ...      249.513 sec
NEB                                    ...      249.513 sec  100.0%


--------------------------------
SUGGESTED CITATIONS FOR THIS RUN
--------------------------------

Below you find a list of papers that are relevant to this ORCA run
We neither can nor want to force you to cite these papers, but we appreciate if you do
You receive ORCA, which is the product of decades of hard work by many enthusiastic individuals, for free
The only thing we kindly ask in return is that you cite our papers,
We deeply appreciate it, if you show your appreciation for ORCA by not just citing the generic ORCA reference.

Please note that relegating all ORCA citations to the supporting information does *not* help us.
SI sections are not indexed - citations you put there will not count into any citation statistics
But we need these citations in order to attract the funding resources that allow us to do what we are doing

Therefore, if you are a happy ORCA user, please consider citing a few of the papers listed below in the main body of your paper

In addition to the list printed below, the program has created the file input.bibtex that contains the list in bibtex format
You can import this file easily into all common literature databanks and citation aid programs


List of essential papers. We consider these as the minimum necessary citations

  1. Neese, F.
     Software update: the ORCA program system, version 6.0
     WIRES Comput. Molec. Sci. 2025 15(1), e70019
     doi.org/10.1002/wcms.7019

List of papers to cite with high priority. The work reported in these papers was absolutely
necessary for this run to complete.
Our perspective: the developers of density functionals and basis sets usually get cited in chemistry papers
Good! But without the algorithms to do something with them, the functionals or basis sets would not do anything.
Hence, in our opinion, the algorithm design and method developments papers are equally worthy of getting cited

  1. Neese, F.
     An improvement of the resolution of the identity approximation for the formation of the Coulomb matrix
     J. Comp. Chem. 2003 24(14), 1740-1747
     doi.org/10.1002/jcc.10318
  2. Asgeirsson, V.; Orri Birgisson, B.; BjÃ¶rnsson, R.; Becker, U.; Neese, F.; Riplinger, C.; Jonsson, H.
     Nudged Elastic Band Method for Molecular Reactions Using Energy-Weighted Springs Combined with Eigenvector Following
     J. Chem. Theory Comput. 2021 17(8), 4929-4945
     doi.org/10.1021/acs.jctc.1c00462
  3. Neese, F.
     The SHARK Integral Generation and Digestion System
     J. Comp. Chem. 2022 44(3), 381
     doi.org/10.1002/jcc.26942
  4. Schmerwitz, Y.L.A.; Asgeirsson, V.; Jonsson, H.
     Improved Initialization of Optimal Path Calculations
     Using Sequential Traversal over the
     Image-Dependent Pair Potential Surface
     J. Chem. Theory Comput. 2023 20(1), 155-163
     doi.org/10.1021/acs.jctc.3c01111

List of suggested additional citations. These are papers that are important in the 'surrounding' of 
of this run, or papers that preceded the highly important papers. If you like your results we are grateful for a citation.

  1. Neese, F.
     The ORCA program system
     WIRES Comput. Molec. Sci. 2012 2(1), 73-78
     doi.org/10.1002/wcms.81
  2. Neese, F.
     Software update: the ORCA program system, version 4.0
     WIRES Comput. Molec. Sci. 2018 8(1), 1-6
     doi.org/10.1002/wcms.1327
  3. Neese, F.; Wennmohs, F.; Becker, U.; Riplinger, C.
     The ORCA quantum chemistry program package
     J. Chem. Phys. 2020 152(22), 224108
     doi.org/10.1063/5.0004608
  4. Neese, F.
     Software update: The ORCA program systemâVersion 5.0
     WIRES Comput. Molec. Sci. 2022 12(1), e1606
     doi.org/10.1002/wcms.1606

List of optional additional citations

  1. Neese, F.
     Approximate second-order SCF convergence for spin unrestricted wavefunctions
     Chem. Phys. Lett. 2000 325(1-3), 93-98
     doi.org/10.1016/s0009-2614(00)00662-x

Timings for individual modules:

Sum of individual times          ...      260.192 sec (=   4.337 min)
Startup calculation              ...        1.892 sec (=   0.032 min)   0.7 %
SCF iterations                   ...        6.851 sec (=   0.114 min)   2.6 %
SCF Gradient evaluation          ...        1.936 sec (=   0.032 min)   0.7 %
NEB Iterations                   ...      249.513 sec (=   4.159 min)  95.9 %
                             ****ORCA TERMINATED NORMALLY****
TOTAL RUN TIME: 0 days 0 hours 4 minutes 11 seconds 581 msec
